actors/miner/src/types.rs

```diff
@@ -293,12 +293,6 @@ pub struct PreCommitSectorParams {
     pub seal_rand_epoch: ChainEpoch,
     pub deal_ids: Vec<DealID>,
     pub expiration: ChainEpoch,
-    /// Deprecated:
-    /// Whether to replace a "committed capacity" no-deal sector (requires non-empty DealIDs)
-    pub replace_capacity: bool,
-    /// Deprecated:
-    /// The committed capacity sector to replace, and its deadline/partition location
-    pub replace_sector_deadline: u64,
```

Comment:
You can't change Params structs. This is a publicly exported API invoked from off-chain. This will break existing callers. We can one day deprecate the entire method, forcing every remaining caller to migrate to PreCommitSectorBatch2, but until then it needs to remain as-is. Deprecated doesn't mean we can delete it – it means we can't delete it but people should stop using it.

Reply:
@anorth For my ow understanding, what kind of offchain callers use this public API ?